GameOfLife/GameOfLife/Conway/ConwayBoard.cs

```diff
@@ -0,0 +1,172 @@
+﻿
+namespace GameOfLife
+{
+    /// <summary>
+    /// Board for Conway's game of life.
+    /// </summary>
+    public class ConwayBoard : Board
+    {
+        /// <summary>
+        /// Count of live cells on the board.
+        /// </summary>
+        private int liveCellCount { get; set; } = 0;
```

Comment:
Add underscore for private properties.
We use underscore to easily distinguish if it is private property or local property.

Reply:
Property name changed.